include/seqan/arg_parse/tool_doc.h

```diff
@@ -874,6 +875,47 @@ inline CharString const & getShortDescription(ToolDoc const & doc)
 }
 
 // --------------------------------------------------------------------------
+// Function setUrl()                                              ToolDoc
+// --------------------------------------------------------------------------
+
+/*!
+ * @fn ToolDoc#setUrl
+ * @headerfile <seqan/arg_parse.h>
+ * @brief Set the url string to the app repository or website.
+ *
+ * @signature void setUrl(toolDoc, str);
+ *
+ * @param[in,out] toolDoc The ToolDoc object to the set the version string for.
+ * @param[in]     str     The url string of the tool (@link CharString @endlink).
+ */
+
+inline void setUrl(ToolDoc & doc, CharString const & url)
+{
+    doc._url = url;
```

Comment:
For consistency, use copy'n'swap.
See: https://github.com/seqan/seqan/pull/1760/files#diff-5576254dc1d9732a6bc7c84aefc8b04dR186